docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -1055,8 +1055,83 @@ Because `MediaLibrary` is the `TvLibrary` class parent, a user with the `admin`
 
 CAUTION: Annotation-based permissions do not work with custom xref:security-customization.adoc#jaxrs-security-context[Jakarta REST SecurityContexts] because there are no permissions in `jakarta.ws.rs.core.SecurityContext`.
 
+[[permission-checker]]
+==== Create permission checkers
+
+Previous examples required that the `SecurityIdentity` possessed required permissions.
+We added these permissions in the `SecurityIdentityAugmentor` and used role-to-permission mapping.
+Another way to check permissions required by the `@PermissionsAllowed` security annotation is a permission checker.
+The permission checkers are CDI bean methods annotated with a `@PermissionChecker`.
+For example, a permission checker can be created like this:
```

Comment:
I feel, given how significant this enhancement is, that we provide more intersting examples instead of say hello 🙂, IMHO, doing an `owner` example would be better.

Reply:
I copied and tweak example in the https://github.com/quarkusio/quarkus/issues/43238 description. I did not include all these classes there as it seems like unnecessary noise. We need to show what the `@PermissionChecker` can do, capabilities, not ideas how to use it. Examples and docs is highly opinioned, let's suggest better example, I am completely fine with it. Thanks